docs/components/Card.md

```diff
@@ -124,4 +124,23 @@ In the layout:
     </LinearLayout>
 
 </com.google.android.material.card.MaterialCardView>
+```
+
+### JetPackCompose
```

Comment:
Je pense qu'il faudrait ajouter un `#` : `#### In Jetpack Compose` et du coup ajouter plus haut un sous titre `#### In XML`

C'est ce que j'ai fait dans la doc de mon côté mais de toute façon il va falloir qu'on se mette d'accord sur un formalisme pour être homogène partout. On reprendra si ça va pas. J'aurais bien aimé pouvoir générer un onglet "XML" et un onglet "Compose", ça se fait en Markdown mais ça marche pas visiblement quand c'est retranscrit en HTML ou j'ai pas trouvé